src/utils/set-state.ts

```diff
@@ -0,0 +1,549 @@
+        }
+    );
+
+const setVaultIssuedTokens = ({ api }: DefaultInterBtcApi) =>
+    disconnectApiOnExit(
+        api,
+        async ({ accountId, collateralSymbol, wrappedSymbol, value }: SetIssuedTokensParams) => {
+            const vaultId = constructVaultId(api, accountId, collateralSymbol, wrappedSymbol);
+
+            const parsedIssuedTokens = api.createType("u128", value);
+
+            const modifier = (vaultData: MutableVaultData): MutableVaultData => ({ ...vaultData, issuedTokens: parsedIssuedTokens });
+
+            console.log(`Setting vault issued tokens to ${value}...`);
+            await modifyVaultData(api, vaultId, modifier);
+            console.log("OK: Succesfully set vault issued tokens.");
+        }
+    );
+
+const setVaultTheft = ({ api }: DefaultInterBtcApi) =>
+    disconnectApiOnExit(
+        api,
+        async ({ accountId, collateralSymbol, wrappedSymbol }: SetVaultParamsBase) => {
+            const vaultId = constructVaultId(api, accountId, collateralSymbol, wrappedSymbol);
+
+            const theftCommittedStatus = { "committedTheft": true } as unknown as VaultRegistryVaultStatus;
+
+            const modifier = (vaultData: MutableVaultData): MutableVaultData => ({ ...vaultData, status: theftCommittedStatus });
+
+            console.log("Setting vault to 'CommittedTheft' status...");
+            await modifyVaultData(api, vaultId, modifier);
+            console.log("OK: Succesfully set vault to 'CommittedTheft' status.");
+        }
+    );
+
+const setVaultBan = (interBtcApi: DefaultInterBtcApi) =>
+    disconnectApiOnExit(
+        interBtcApi.api,
+        async ({ accountId, collateralSymbol, wrappedSymbol }: SetVaultParamsBase) => {
+            const { api } = interBtcApi;
+            const vaultId = constructVaultId(api, accountId, collateralSymbol, wrappedSymbol);
+            const currentBlockNumber = await interBtcApi.system.getCurrentBlockNumber();
+
+            // Bans vault for the next 1000 blocks
+            const newBannedUntil = api.createType("Option<u32>", currentBlockNumber + 1000);
+
+            const modifier = (vaultData: MutableVaultData): MutableVaultData => ({ ...vaultData, bannedUntil: newBannedUntil });
+
+            console.log("Setting vault to be banned for the next 1,000 blocks...");
+            await modifyVaultData(api, vaultId, modifier);
+            console.log(`OK: Succesfully banned vault until block ${newBannedUntil}.`);
+        }
+    );
+
+const setVaultUnban = ({ api }: DefaultInterBtcApi) =>
+    disconnectApiOnExit(
+        api,
+        async ({ accountId, collateralSymbol, wrappedSymbol }: SetVaultParamsBase) => {
+            const vaultId = constructVaultId(api, accountId, collateralSymbol, wrappedSymbol);
+
+            const newBannedUntil = null;
+
+            const modifier = (vaultData: MutableVaultData): MutableVaultData => ({ ...vaultData, bannedUntil: <any>newBannedUntil });
```

Comment:
Not sure if we can get rid of the warning here (Unexpected any), but if not, please add ignore line above.